[CI] CsvIT test {csv-spec:promql-native-histogram-support.histogram_sum_increase_exp_histo_all_instances} failing

**Build Scans:**
- [elasticsearch-periodic #12581 / openjdk26_checkpart3_java-matrix](https://gradle-enterprise.elastic.co/s/t6uc773yrvmwa)

**Reproduction Line:**
```
./gradlew ":x-pack:plugin:esql:internalClusterTest" --tests "org.elasticsearch.xpack.esql.CsvIT" -Dtests.method="test {csv-spec:promql-native-histogram-support.histogram_sum_increase_exp_histo_all_instances}" -Dtests.seed=30EC64C2C997B3F8 -Dtests.locale=lkt-Latn-US -Dtests.timezone=Asia/Ulaanbaatar -Druntime.java=26
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.xpack.esql.CsvIT),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!('test%20%7Bcsv-spec%3Apromql-native-histogram-support.histogram_sum_increase_exp_histo_all_instances%7D'),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: 
Data mismatch:
row 0 column 0:a list containing
row 0 column 0:0: expected <1472.744209000001> but was <1472.7442090000013>
Actual:
result:double      | step:datetime            | _timeseries:keyword
1472.7442090000013 | 2025-09-25T02:00:00.000Z | {"instance":"instance-0"}
36.19848400000001  | 2025-09-25T02:00:00.000Z | {"instance":"instance-1"}
27.706021000000007 | 2025-09-25T02:00:00.000Z | {"instance":"instance-2"}

Expected:
result:double      | step:datetime            | _timeseries:keyword
1472.744209000001  | 2025-09-25T02:00:00.000Z | {"instance":"instance-0"}
36.19848400000001  | 2025-09-25T02:00:00.000Z | {"instance":"instance-1"}
27.706021000000007 | 2025-09-25T02:00:00.000Z | {"instance":"instance-2"}

```

**Issue Reasons:**
- [main] 1 consecutive failures in test test {csv-spec:promql-native-histogram-support.histogram_sum_increase_exp_histo_all_instances}
- [main] 1 consecutive failures in step openjdk26_checkpart3_java-matrix
- [main] 1 consecutive failures in pipeline elasticsearch-periodic

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-core-infra.

:eyes:

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -529,12 +529,6 @@ tests:
 - class: org.elasticsearch.compute.operator.topn.GroupedTopNOperatorTests
   method: testRandomMultipleColumns
   issue: https://github.com/elastic/elasticsearch/issues/155019
-- class: org.elasticsearch.xpack.esql.CsvIT
-  method: test {csv-spec:promql-native-histogram-support.histogram_sum_increase_exp_histo_all_instances}
-  issue: https://github.com/elastic/elasticsearch/issues/155023
-- class: org.elasticsearch.xpack.esql.CsvIT
-  method: test {csv-spec:promql-native-histogram-support.histogram_avg_increase_exp_histo_all_instances}
-  issue: https://github.com/elastic/elasticsearch/issues/155024
 - class: org.elasticsearch.xpack.inference.integration.SemanticIndexOptionsIT
   method: testValidateIndexOptionsWithBasicLicense
   issue: https://github.com/elastic/elasticsearch/issues/155026

--- a/x-pack/plugin/esql/qa/testFixtures/src/main/resources/promql-native-histogram-support.csv-spec
+++ b/x-pack/plugin/esql/qa/testFixtures/src/main/resources/promql-native-histogram-support.csv-spec
@@ -141,9 +141,9 @@ PROMQL index=exp_histo_sample step=2h start="2025-09-25T02:00:00.000Z" end="2025
 | SORT _timeseries;
 
 result:double              | step:datetime            | _timeseries:keyword
-1472.744209..1472.744210   | 2025-09-25T02:00:00.000Z | "{""instance"":""instance-0""}"
-36.198484..36.198485       | 2025-09-25T02:00:00.000Z | "{""instance"":""instance-1""}"
-27.706021..27.706022       | 2025-09-25T02:00:00.000Z | "{""instance"":""instance-2""}"
+1472.74..1472.75           | 2025-09-25T02:00:00.000Z | "{""instance"":""instance-0""}"
+36.19..36.20               | 2025-09-25T02:00:00.000Z | "{""instance"":""instance-1""}"
+27.70..27.71               | 2025-09-25T02:00:00.000Z | "{""instance"":""instance-2""}"
 ;
 
 ///////////////////////////////////////////////////////////////////////////////
@@ -164,9 +164,9 @@ PROMQL index=exp_histo_sample step=2h start="2025-09-25T02:00:00.000Z" end="2025
 | SORT _timeseries;
 
 result:double                | step:datetime            | _timeseries:keyword
-0.166581179..0.166581180     | 2025-09-25T02:00:00.000Z | "{""instance"":""instance-0""}"
-0.011137995..0.011137996     | 2025-09-25T02:00:00.000Z | "{""instance"":""instance-1""}"
-0.008197047..0.008197048     | 2025-09-25T02:00:00.000Z | "{""instance"":""instance-2""}"
+0.16..0.17                   | 2025-09-25T02:00:00.000Z | "{""instance"":""instance-0""}"
+0.01..0.02                   | 2025-09-25T02:00:00.000Z | "{""instance"":""instance-1""}"
+0.00..0.01                   | 2025-09-25T02:00:00.000Z | "{""instance"":""instance-2""}"
 ;
 
 histogram_count_aggsum_increase_exp_histo

--- a/x-pack/plugin/esql/qa/testFixtures/src/main/resources/promql-native-histogram-support.csv-spec
+++ b/x-pack/plugin/esql/qa/testFixtures/src/main/resources/promql-native-histogram-support.csv-spec
@@ -140,10 +140,10 @@ required_capability: promql_increase_on_histogram
 PROMQL index=exp_histo_sample step=2h start="2025-09-25T02:00:00.000Z" end="2025-09-25T02:00:00.000Z" result=(histogram_sum(increase(responseTime{instance=~"instance-.*"}[2h])))
 | SORT _timeseries;
 
-result:double      | step:datetime            | _timeseries:keyword
-1472.744209000001  | 2025-09-25T02:00:00.000Z | "{""instance"":""instance-0""}"
-36.19848400000001  | 2025-09-25T02:00:00.000Z | "{""instance"":""instance-1""}"
-27.706021000000007 | 2025-09-25T02:00:00.000Z | "{""instance"":""instance-2""}"
+result:double              | step:datetime            | _timeseries:keyword
+1472.744209..1472.744210   | 2025-09-25T02:00:00.000Z | "{""instance"":""instance-0""}"
+36.198484..36.198485       | 2025-09-25T02:00:00.000Z | "{""instance"":""instance-1""}"
+27.706021..27.706022       | 2025-09-25T02:00:00.000Z | "{""instance"":""instance-2""}"
 ;
 
 ///////////////////////////////////////////////////////////////////////////////
@@ -163,10 +163,10 @@ required_capability: promql_increase_on_histogram
 PROMQL index=exp_histo_sample step=2h start="2025-09-25T02:00:00.000Z" end="2025-09-25T02:00:00.000Z" result=(histogram_avg(increase(responseTime{instance=~"instance-.*"}[2h])))
 | SORT _timeseries;
 
-result:double        | step:datetime            | _timeseries:keyword
-0.16658117961769042  | 2025-09-25T02:00:00.000Z | "{""instance"":""instance-0""}"
-0.011137995076923079 | 2025-09-25T02:00:00.000Z | "{""instance"":""instance-1""}"
-0.008197047633136096 | 2025-09-25T02:00:00.000Z | "{""instance"":""instance-2""}"
+result:double                | step:datetime            | _timeseries:keyword
+0.166581179..0.166581180     | 2025-09-25T02:00:00.000Z | "{""instance"":""instance-0""}"
+0.011137995..0.011137996     | 2025-09-25T02:00:00.000Z | "{""instance"":""instance-1""}"
+0.008197047..0.008197048     | 2025-09-25T02:00:00.000Z | "{""instance"":""instance-2""}"
 ;
 
 histogram_count_aggsum_increase_exp_histo

